packages/fxa-payments-server/src/components/PaymentErrorView/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Localized } from '@fluent/react';
+import { StripeError } from '@stripe/stripe-js';
+
+import { getErrorMessage } from '../../lib/errors';
+import errorIcon from '../../images/error.svg';
+import SubscriptionTitle from '../SubscriptionTitle';
+
+import './index.scss';
+
+type GeneralError = {
+  code: string;
+};
+
+export type PaymentErrorViewProps = {
+  error: undefined | StripeError | GeneralError;
```

Comment:
Could this be an optional prop instead of having a value of `undefined`?

Reply:
Yes, I copied over the type from the other view, optional is a cleaner way to write it.